integration_tests/dataproc_test_case.py

```diff
@@ -159,14 +167,19 @@ def createCluster(self,
         args.append("--master-boot-disk-size={}".format(boot_disk_size))
         args.append("--worker-boot-disk-size={}".format(boot_disk_size))
 
+        args.append("--master-boot-disk-type=pd-ssd")
+        args.append("--worker-boot-disk-type=pd-ssd")
+
         args.append("--format=json")
 
         args.append("--region={}".format(self.REGION))
         if self.cluster_zone:
           args.append("--zone={}".format(self.cluster_zone))
 
         if not FLAGS.skip_cleanup:
-            args.append("--max-age=2h")
+            args.append("--max-age=30m")
```

Comment:
How long does GPU tests take to finish? I thought the cluster creation itself takes lot of time around 30m.

Reply:
10-16 minutes ; but I'm either merging these changes into Prince's changes or discarding them.  I'm discarding this one, but would keep it in gpu/test_gpu.py if I could.

Reply:
oh, I see what you're saying.  The tests don't take long to run.  The delays come from gcloud compute ssh mostly.  Once we turn the MIG tests back on, we need to account for nvidia-smi being super slow on systems with 8 attached H100s.  The commands are simple and should not take more than 30s to execute each.

With the longest measured duration of fetching, building, installing and signing kernel modules and installing about 10GB of binary support libraries taking about 16 minutes, plus 30s for each test run, ~10, then we have 21 minutes.  We could try reducing it to 25m and see if all tests pass, but I think leaving five minutes of headroom for congested networks or retries will give us a more reliable pass rate

Reply:
Here are stats from the most recent run

```
Step #2 - "dataproc-2.0-debian10-tests": 2024-08-02T19:27:53.262890610Z   Stats over 15 runs: max = 2313.8s, min = 24.8s, avg = 1021.0s, dev = 592.2s
Step #3 - "dataproc-2.0-rocky8-tests": 2024-08-02T18:58:06.475658471Z   Stats over 15 runs: max = 376.8s, min = 141.2s, avg = 297.8s, dev = 100.6s
Step #5 - "dataproc-2.1-debian11-tests": 2024-08-02T19:29:09.794418578Z //gpu:test_gpu                                                           PASSED in 2401.5s
Step #6 - "dataproc-2.1-rocky8-tests": 2024-08-02T18:55:34.482352277Z //gpu:test_gpu                                                           PASSED in 268.7s
Step #7 - "dataproc-2.1-ubuntu20-tests": 2024-08-02T19:28:33.297155580Z   Stats over 15 runs: max = 2473.7s, min = 116.1s, avg = 1005.6s, dev = 628.7s
Step #8 - "dataproc-2.2-debian12-tests": 2024-08-02T19:19:00.625770554Z   Stats over 15 runs: max = 1917.9s, min = 116.4s, avg = 897.4s, dev = 535.6s
Step #9 - "dataproc-2.2-rocky9-tests": 2024-08-02T18:55:36.599559802Z //gpu:test_gpu                                                           PASSED in 264.5s
```